contributors/design-proposals/node/kubelet-cri-logging.md

```diff
@@ -161,7 +161,7 @@ For example,
 ```
 
 With the knowledge, kubelet can parses the logs and serve them for `kubectl
-logs` requests. This meets requirement (3). Note that the format is defined
+logs` requests. This meets requirement (1). Note that the format is defined
```

Comment:
As kubelet can parse the log, so it can serve the function "kubectl log", which is more suitable for requirement 1 "Provide ways for CRI-compliant runtimes to support all existing logging features, i.e., kubectl logs.".